sci-chemistry/coordgenlibs/metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
```

Comment:
It would perhaps be more prudent to drop support for maeparser/coordgenlibs.  They are only optional dependencies that can always be added later if sci-chemistry wants it.